homeassistant/components/tts/__init__.py

```diff
@@ -415,6 +427,18 @@ def async_get_supported_voices(self, language: str) -> list[Voice] | None:
     async def async_internal_added_to_hass(self) -> None:
         """Call when the entity is added to hass."""
         await super().async_internal_added_to_hass()
+        try:
+            _ = self.default_language
```

Comment:
I don't think the assignment is needed?

Reply:
It is needed because else it is considered a useless expression.